fedimint-testing/src/fixtures.rs

```diff
@@ -113,6 +113,19 @@ impl Fixtures {
         self
     }
 
+    pub fn with_server_only_module(
```

Comment:
Would this make sense to use for other fixtures instead of only ln-gateway?

Reply:
Yes, but with very diminishing returns.